src/itemActionTitle/ItemActionTitle.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import cc from 'classcat'
+
+import { Item } from '../_internals/item'
+import { color } from '../_utils/branding'
+import { useFocusVisible } from '../_utils/focusVisibleProvider/useFocusVisible'
+import { A11yProps, pickA11yProps } from '../_utils/interfaces'
+import { NormalizeProps } from '../layout/layoutNormalizer'
+
+export type ItemActionTitleProps = NormalizeProps &
+  A11yProps &
+  Readonly<{
+    className?: string
+    href?: string | JSX.Element
+    titleLabel?: string
+    subLabel?: string
+    action?: string
+    leftAddon?: React.ReactNode
+    onClick?: (event: React.MouseEvent<HTMLElement>) => void
+    onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void
+  }>
+
+export const ItemActionTitle = (props: ItemActionTitleProps) => {
+  const {
+    className,
+    titleLabel,
+    subLabel,
+    action,
+    leftAddon,
+    href,
+    onClick,
+    onMouseDown,
+    hasHorizontalSpacing = false,
+  } = props
+  const a11yAttrs = pickA11yProps<ItemActionTitleProps>(props)
+  const { focusVisible, onFocus, onBlur } = useFocusVisible()
+  return (
+    <Item
+      className={cc([
+        {
+          'focus-visible': focusVisible,
+        },
+        className,
+      ])}
+      leftAddon={leftAddon}
+      leftTitle={titleLabel}
+      leftBody={subLabel}
+      rightBody={action}
+      rightBodyColor={color.blue}
+      href={href}
+      onClick={onClick}
+      onBlur={onBlur}
+      onFocus={onFocus}
+      onMouseDown={onMouseDown}
+      tag={<button type="button" />}
```

Comment:
I think we should have this default tag only when we have in onClick props, but not when we have an href. To avoid the `<a type="button" ...>` issue.

Reply:
You right, I only added it if no `href` prop.